src/TorchSharp/NN/Linear.cs

```diff
@@ -10,47 +10,43 @@ public Linear(IntPtr handle) : base(handle)
         {
         }
 
-        [DllImport("LibTorchSharp")]
-        private static extern IntPtr THSNN_linearModule(long input_size, long output_size, bool with_bias);
+        [DllImport("libTorchSharp")]
```

Comment:
The errors you get for Linux are probably due to the lower case of libTorchSharp.
Linux is case sensitive, and I think I used LibTorchSharp everywhere else in the code.

Let me know if that does not work.

Reply:
Ah-ha! Let me fix this